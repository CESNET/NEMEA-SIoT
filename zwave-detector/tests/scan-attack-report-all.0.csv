time,time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2020-01-19T22:57:00,2020-01-19T22:57:00.000,2,00
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730201010c01720564
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730201010c01861287
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730201010c01010113
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730201010c01200330
2020-01-19T22:57:02,2020-01-19T22:57:02.000,2,e14534730201010d0170060165
