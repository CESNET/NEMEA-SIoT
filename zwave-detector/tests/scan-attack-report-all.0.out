2,2020-01-19T22:57:02.000,1,"Scanned: Device Manufacturer and Device Type, Device Software Version, Basic Operational Status, Configuration Settings,"
